src/ride/views.py

```diff
@@ -1 +1,44 @@
-# Create your views here.
+import json
+
+from api import settings as api_settings
+from api.utils import failure_response
+from api.utils import success_response
+from rest_framework import generics
+from rest_framework import status
+
+from .controllers.create_ride_controller import CreateRideController
+from .models import Ride
+from .serializers import RideSerializer
+
+
+class AllRidesView(generics.GenericAPIView):
+    serializer_class = RideSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def get(self, request):
+        """Get all rides."""
+        rides = [self.serializer_class(ride).data for ride in Ride.objects.all()]
+        return success_response(rides)
```

Comment:
```
return success_response(self.serializer_class(Ride.objects.all(),many=True))
```

Reply:
double check that this works tho

Reply:
i get `TypeError: Object of type ListSerializer is not JSON serializable` error :(